packages/e2e-tests/src/specs/dashboard/settings/adminUser/customFonts.js

```diff
@@ -99,16 +96,6 @@ describe('Custom Fonts', () => {
     await takeSnapshot(page, 'Custom Fonts Settings');
   });
 
-  it('should select font when clicked', async () => {
```

Comment:
This should be replaced with a Jest test vs e2e.

Reply:
Let's create a ticket for it

Reply:
https://github.com/GoogleForCreators/web-stories-wp/issues/11929